types/prompts.ts

```diff
@@ -9,11 +11,27 @@ type PromptType =
   | 'checkbox'
   | 'input'
   | 'password'
+  | 'number'
   | 'rawlist';
 
-export type PromptChoices =
-  | string[]
-  | Array<{ name: string; value: string | { [key: string]: string } }>;
+export type PromptChoices = Array<
+  | string
+  | {
+      name: string;
+      value?:
```

Comment:
Can probably just make `value` `any` at this point

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1293/commits/64ec64e3b160db98474cf96fff85513b85b11095